src/pages/Watchlist/List/component.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Row,
+  Col,
+  Pagination,
+  Empty,
+  Typography,
+  Spin
+} from 'antd';
+
+import Movie from '../../../components/MovieItem';
+
+const List = ({
```

Comment:
I think this component should be reused with dashboard list, because them have similar functionality. You already have all business logic in the Watchlist component, so it should be not difficult to adapt universal movie list component

Reply:
Done